README.md

```diff
@@ -109,9 +109,6 @@ security:
         default:
```

Comment:
do we even need to define the default encoder? I think Symfony uses bcrypt if nothing is defined automatically.

Reply:
I'm not sure.

Reply:
`->scalarNode('algorithm')->cannotBeEmpty()->end()`

Reply:
I digged some code. `default` here only works because our user object implements `EncoderAwareInterface` and returns this value. The _correct_ approach would be to define the encoder by class name, so it should look like this:

```
security:
    encoders:
        Contao\User:
            algorithm: bcrypt
```

Reply:
Changed in 01bb96ee64808027bb6bc15d5cdc02911d5f9d8b.